Source/Core/DolphinQt/Config/Mapping/IOWindow.h

```diff
@@ -94,4 +99,18 @@ class IOWindow final : public QDialog
 
   ciface::Core::DeviceQualifier m_devq;
   Type m_type;
+  std::shared_ptr<ciface::Core::Device> m_selected_device;
+};
+
+class InputStateDelegate : public QItemDelegate
```

Comment:
This class could be moved (along with the header above) to the cpp

Reply:
Done.